### PR TITLE
Fix macros completion e2e test

### PR DESCRIPTION
**Pull request by fivetran-dmitryboykov**

### CODE REVIEW

File: e2e/src/completion/completion_macros.spec.ts [hunk 1]
```diff
@@ -12,20 +12,17 @@ suite('Should suggest macros completions', () => {
...
 
     // act
-
-    // new Position(0, 17) -- ?
-    const actualCompletionList = await triggerCompletion(docUri, new Position(0, 15), 'e');
+    const actualCompletionList = await triggerCompletion(docUri, new Position(0, 17));
 
     // assert
     const expectedCompletions = getMacrosCompletionList();
-    assertThat(actualCompletionList.items.length, expectedCompletions.length);
```
Comment (pgrivachev) [inline e2e/src/completion/completion_macros.spec.ts:28]: The problem of this test also that if we return `undefined`, the default set of suggestions contains expected values `extract_first_name` and `extract_last_name`. Can we make it more specific somehow?

**Review by pgrivachev — APPROVED** [review]
Looks good, just need to think is we can avoid false positive in test.

File: e2e/src/completion/completion_macros.spec.ts (hunk 1 above)
Comment (fivetran-dmitryboykov) [reply e2e/src/completion/completion_macros.spec.ts:28]: Completion provider returns items with detail property equal to 'Macro'. Added detail property comparison.

Comment (fivetran-dmitryboykov) [reply e2e/src/completion/completion_macros.spec.ts:28]: The same for other autocompletion tests

**Pull request merged**